### PR TITLE
Convert facets to aggregations (bug 1116800)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: mkt/stats/views.py [hunk 1]
```diff
@@ -299,15 +299,15 @@ def get_query(self, metric, field, app_id=None):
...
+            count = agg.get('count', 0)
 
-            # We filter out facets with count=0 to avoid returning things
+            # We filter out aggs with count=0 to avoid returning things
             # like `'max': u'-Infinity'`.
             if count > 0:
-                for field in ('max', 'mean', 'min', 'std_deviation',
-                              'sum_of_squares', 'total', 'variance'):
-                    value = facet.get(field)
+                for field in ('avg', 'max', 'min', 'std_deviation',
```
Comment (diox) [inline mkt/stats/views.py]: We're exposing these fields directly in the API and one of them (`mean`) is no longer present, isn't that going to be a problem ?

Comment (diox) [inline mkt/stats/views.py]: Also, tests were not covering this ?

Comment (robhudson) [inline mkt/stats/views.py]: Yeah, I couldn't find any tests other than to test the status codes and permission stuff. I could rename the changed fields so it remains consistent. (And add a test?) This was just a quick work at the end of the day.

**Comment by ngokevin** [comment]
Going to close for age.

**Pull request closed without merging**